src/pages/editor/PublicationChannelTableRow.tsx

```diff
@@ -0,0 +1,37 @@
+import { Chip, TableCell, TableRow } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useFetchOrganization } from '../../api/hooks/useFetchOrganization';
+import { useFetchPublisher } from '../../api/hooks/useFetchPublisher';
+import { ClaimedChannel } from '../../types/customerInstitution.types';
+import { getIdentifierFromId } from '../../utils/general-helpers';
+import { getLanguageString } from '../../utils/translation-helpers';
+
+interface ChannelClaimTableRowProps {
+  channel: ClaimedChannel;
+}
+export const ChannelClaimTableRow = ({ channel: claim }: ChannelClaimTableRowProps) => {
```

Comment:
ah, ser ut som det brukes via annet navn her uansett. Da bør hvertfall `channel` over her renames
```suggestion
export const ChannelClaimTableRow = ({ claim }: ChannelClaimTableRowProps) => {
```